cmake/Windows/build-imagemagick.sh

```diff
@@ -150,8 +150,9 @@ function buildlibxml2() {
     fi
     extractIfNeeded libxml2-*.tar.gz
     cd libxml2-*/win32/
+    ised configure.js 's/dirSep = "\\\\";/dirSep = "\/";/'
     cscript.exe configure.js compiler=mingw prefix=/usr/local
-    ised ../dict.c '/typedef.*uint32_t;$/d'
+    # ised ../dict.c '/typedef.*uint32_t;$/d'
```

Comment:
With this line the compilation failed because the compiled didn't find `uint32_t`